classes/plugins/Plugin.inc.php

```diff
@@ -312,50 +314,47 @@ function getPluginPath() {
 	}
 
 	/**
-	 * Return the Resource Name for templates in this plugin.
-	 *
+	 * Return the Resource Name for templates in this plugin, or if specified, the full resource locator
+	 * for a specific template.
+	 * @param $template Template path/filename, if desired
 	 * @return string
 	 */
-	public function getTemplateResourceName($inCore = false) {
+	public function getTemplateResource($template = null, $inCore = false) {
```

Comment:
It was already missing, but `$inCore` isn't documented here.

Reply:
Thanks, fixed.